lib/ast.js

```diff
@@ -1241,6 +1242,21 @@ var AST_True = DEFNODE("True", null, {
     value: true
 }, AST_Boolean);
 
+/* -----[ Yield ]----- */
+
+var AST_Yield = DEFNODE("Yield", "value is_star", {
+    $documentation: "A `yield` statement",
+    $propdoc: {
+        value: "[AST_Node?] the value returned or thrown by this statement; could be null",
+        is_star: "[Boolean] Whether this is a yield or yield* statement"
+    },
+    _walk: function(visitor) {
+        return visitor._visit(this, this.value && function(){
+            this.value._walk(visitor);
+        });
+    }
+}, AST_Statement);
```

Comment:
Looking at your tests... I didn't realize that `yield` could be used in an expression context. If that's the case then it should not be an AST_Statement - or derived from anything for that matter. Line 1258 should be:
```
});
```
My apologies - this is the 3rd or 4th time I suggested the base class for `AST_Yield` be changed.